src/routing_membrane.rs

```diff
@@ -340,6 +352,36 @@ impl<F> RoutingMembrane<F> where F: Interface {
         // TODO: trigger churn on boolean
     }
 
+    /// Parse and update the header with us as a relay node;
```

Comment:
I agree it is not necessary; I introduced it as a logical-debug-tool;  I wanted to make sure that I could not by accident pass a relay message into the SAFE network without the compiler complaining.  in L206 we tag the name with its source (ie a message that was already on the SAFE network, or a to-be relayed message/to-be introduced onto the SAFE network by the relay node